### ISSUE TITLE
Colors in config.toml not overwriting

### ISSUE THREAD

**Issue by csajedi** [okkur/syna#581]
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug 

**What happened**:
When I use the syna starter repo I change the colors from the config.toml file but they are not updated in fact on my webpage. Even after server restarts.
**What you expected to happen**:
The params.colors that I set should update the fragments that have a background color setting. This color stays the old preset color regardless. 

**How to reproduce it (as minimally and precisely as possible)**:
Use the starter repo and try to modify the colors set in the config.toml.

**Anything else we need to know?**:

**Environment**: Linux (Ubuntu LTS)
- Syna Theme version:  v0.15.0
- Hugo version:  v0.55.6-A5D4C82D
- Others:

**Comment by stp-ip** [comment]
Try running the hugo-extended version. This should rewrite the specific css files necessary.

**Comment by csajedi** [comment]
I struggled to get hugo-extended working locally so I'm using netlify with it configured and it all works. Closing issue.

**Comment by SenoOtway** [comment]
> Try running the hugo-extended version. This should rewrite the specific css files necessary.

Very valuable tip. Since I just delved into building sites with Hugo, I was not aware of that and also had no idea what to search for here in the issues. I was on the verge of abandoning the Syna theme, when I found the hint to watch out for the hugo-extended binary in the documentation of another theme using sass.

Thus, I would advocate to include a small hint in the documention, e.g. on the page "Supported Colors".

Thanks for the otherwise really great theme!
